pkg/templates/config_context.go

```diff
@@ -65,6 +74,7 @@ func (ctx ConfigCtx) FuncMap() template.FuncMap {
 		"ConfigOptionData":      ctx.configOptionData,
 		"ConfigOptionEquals":    ctx.configOptionEquals,
 		"ConfigOptionNotEquals": ctx.configOptionNotEquals,
+		"KubeSeal":              ctx.kubeSeal,
```

Comment:
It seems a little unexpected to me that this exists on ConfigContext and not on StaticContext -- does it explicitly rely on config options?